include/wx/uilocale.h

```diff
@@ -138,11 +138,68 @@ class WXDLLIMPEXP_BASE wxUILocale
     // its dtor is not virtual.
     ~wxUILocale();
 
+    // Try to get user's (or OS's) preferred language setting.
+    // Return wxLANGUAGE_UNKNOWN if language-guessing algorithm failed
+    static int GetSystemLanguage();
+
+    // get the encoding used by default for text on this system, returns
+    // wxFONTENCODING_SYSTEM if it couldn't be determined
+    static wxFontEncoding GetSystemEncoding();
+
+    // get the string describing the system encoding, return empty string if
+    // couldn't be determined
+    static wxString GetSystemEncodingName();
```

Comment:
We probably don't need this one neither.

Reply:
Ok, I will revert this change, too.